### DISCUSSION TITLE
OSS Weather is now on AlternativeTo

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#121] [Show and tell]
I went ahead and submitted OSS Weather to AlternativeTo.net. 
https://alternativeto.net/software/oss-weather/about/

I recommend giving it a like/star to help it boost its ranking and bring new users. Also, you're welcome to update information and recommend apps for its page.

**Comment by farfromrefug** [comment]
@jacobgonzales20 awesome that KS a lot !
